### PR TITLE
Feature/packages/blog author #cmx 134

### PR DESCRIPTION
**Pull request by Mar0cas**
Blog Author component creation, all the necessary files + logic + tests.

### CODE REVIEW

File: packages/organisms/blog-author/src/component/Defaults.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DefaultProps } from './Types';
+
+const Defaults: DefaultProps = {
+  title: 'Title',
```
Comment (ricardo-valente) [inline packages/organisms/blog-author/src/component/Defaults.ts:4]: I believe this section doesn't have `title` prop

File: packages/organisms/blog-author/src/__tests__/__snapshots__/BlogAuthor.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  data-layout="blog_author"
+>
+  <div
+    class="container"
+  >
+    <div
+      class="row"
+    >
+      <div
+        class="wrapper wrapper--full wrapper--transparent -primary section__blog-author"
```
Comment (ricardo-valente) [inline packages/organisms/blog-author/src/__tests__/__snapshots__/BlogAuthor.test.tsx.snap:15]: `wrapper--full` does not exist, only `normal` or `big`

Comment (Mar0cas) [reply packages/organisms/blog-author/src/__tests__/__snapshots__/BlogAuthor.test.tsx.snap:15]: We need to fix this on Section component

File: packages/organisms/blog-author/src/component/Defaults.ts (hunk 1 above)
Comment (Mar0cas) [reply packages/organisms/blog-author/src/component/Defaults.ts:4]: Removed

**Pull request merged**